<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataRiverPathPointMoveDialog</class>
 <widget class="QDialog" name="GeoDataRiverPathPointMoveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Move Traversal Line</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="CoordinateEditWidget" name="coordX"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="centerCommaLabel">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="CoordinateEditWidget" name="coordY"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="offsetModeRadioButton">
       <property name="text">
        <string>Transfer Quantity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="RealNumberEditWidget" name="offsetX"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="offsetCommaLabel">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="RealNumberEditWidget" name="offsetY"/>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="centerPointModeRadioButton">
       <property name="text">
        <string>Center Point Coordinates</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/realnumbereditwidget.h</header>
  </customwidget>
  <customwidget>
   <class>CoordinateEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/coordinateeditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDataRiverPathPointMoveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDataRiverPathPointMoveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>centerPointModeRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>coordX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>centerPointModeRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>coordY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetModeRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>offsetX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetModeRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>offsetY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
